### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by idamir**
Hello,
I stated works on the Russian translation for libqalculate and qalc.
po/ru.po is ~100% complete,
po-defs/ru.po is ~93% complete.

The translation will be checked with people from US, UK and from form USSR

### CODE REVIEW

File: po-defs/ru.po [hunk 1]
```diff
@@ -0,0 +1,11408 @@
...
+msgid "a-cr:EUR,au:&#x20AC;,euro,p:euros"
+msgstr "a-c:EUR,au:&#x20AC;,евро"
+
+#: ../data/currencies.xml.in.h:4
+msgid ""
+"Andorra, Austria, Belgium, Cyprus, Estonia, Finland, France, Germany, "
+"Greece, Ireland, Italy, Latvia, Lithuania, Luxembourg, Malta, Netherlands, "
+"Portugal, San Marino, Slovakia, Slovenia, Spain, Vatican City"
+msgstr ""
+"Андорра, Австрия, Бельгия, Кипр, Эстония, Финляндия, Франция, Германия, "
```
Comment (i-ky) [inline po-defs/ru.po:39]: Shouldn't this list be alphabetically sorted?

File: po-defs/ru.po [hunk 2]
```diff
@@ -0,0 +1,11408 @@
...
+msgid "Comorian Franc"
+msgstr "Коморский франк"
+
+#: ../data/currencies.xml.in.h:195
+msgid "a-cr:KMF"
+msgstr "a-c:KMF"
+
+#: ../data/currencies.xml.in.h:196
+msgid "Comoros"
+msgstr "Коморские острова "
```
Comment (i-ky) [inline po-defs/ru.po:823]: Extra space.

File: po-defs/ru.po [hunk 3]
```diff
@@ -0,0 +1,11408 @@
...
+msgid ""
+"Calculates the inverse of a matrix. The inverse is the matrix that "
+"multiplied by the original matrix equals the identity matrix (AB = BA = I)."
+msgstr ""
+"Вычисляет обратную матрицу. Обратной является матрица, умножение которой на "
+"исходную матрицу равняется единичной матрице (AB = BA = I). "
+
+#: ../data/functions.xml.in.h:97
+msgid "Load CSV File"
+msgstr "Загрузить файл типа CSV"
```
Comment (i-ky) [inline po-defs/ru.po:3135]: I would use "формат файла" instead of "тип файла".

File: po-defs/ru.po [hunk 4]
```diff
@@ -0,0 +1,11408 @@
...
+msgid "r:magnitude"
+msgstr "r:модуль"
+
+#: ../data/functions.xml.in.h:108
+msgid ""
+"Calculates the magnitude of a value. This function returns the same value as "
+"abs() for all values except vectors."
+msgstr ""
+"Вычисляет значение по модулю или модуль вектора. Эта функция возвращает то "
+"же значение, что и abs(), кроме векторов."
```
Comment (i-ky) [inline po-defs/ru.po:3183]: Something was lost in translation...
```suggestion
"же значение, что и abs(), для всех значений кроме векторов."
```

File: po-defs/ru.po [hunk 5]
```diff
@@ -0,0 +1,11408 @@
...
+"vector 1 and the corresponding (in the same row and column) elements in "
+"matrix/vector 2. An unlimited number of matrices/vectors can be specified, "
+"with each matrix/vector argument followed by the corresponding variable used "
+"in the function argument."
+msgstr ""
+"Вычислят новую матрицу или вектор, используя каждый отдельный элемент в "
+"матрице/векторе 1 и соответствующие (в той же строке и столбце) элементы в "
+"матрице/векторе 2. Можно указать неограниченное количество матриц/векторов с "
+"указанием используемой в аргументе функции имени соответствующей переменной "
+"после каждого аргумента матрицы/вектора.."
```
Comment (i-ky) [inline po-defs/ru.po]: Suggestion:
```suggestion
"указанием имени переменной, используемой в аргументе соответствующей функции, "
"после каждого аргумента матрицы/вектора.."
```

File: po-defs/ru.po [hunk 6]
```diff
@@ -0,0 +1,11408 @@
...
+msgstr "r:факториал2"
+
+#: ../data/functions.xml.in.h:135
+msgid ""
+"Calculates the double factorial of an integer. Multiplies the argument with "
+"every second lesser positive integer (n(n-2)(n-4)...). Can also be entered "
+"as a number followed by two exclamation marks."
+msgstr ""
+"Вычисляет двойной факториал целого числа. Умножает аргумент на каждое "
+"уменьшенной на 2 положительное целое число (n (n-2) (n-3)...). Также можно "
```
Comment (i-ky) [inline po-defs/ru.po:3316]:
```suggestion
"уменьшенное на 2 положительное целое число (n (n-2) (n-4)...). Также можно "
```

File: po-defs/ru.po [hunk 7]
```diff
@@ -0,0 +1,11408 @@
...
+msgstr "r:мультифакториал"
+
+#: ../data/functions.xml.in.h:138
+msgid ""
+"Calculates the multifactorial of an integer. Multiplies the argument with "
+"every x lesser positive integer (n(n-x)(n-2x)...). Can also be entered as a "
+"number followed by three or more exclamation marks."
+msgstr ""
+"Вычисляет k-мультифакториал целого числа. Умножает аргумент на каждое "
+"уменьшенной на k положительное целое число (n (n-k) (n-2k)...). Также можно "
```
Comment (i-ky) [inline po-defs/ru.po:3334]:
```suggestion
"уменьшенное на k положительное целое число (n (n-k) (n-2k)...). Также можно "
```

File: po-defs/ru.po [hunk 8]
```diff
@@ -0,0 +1,11408 @@
...
+msgstr "r:факториал2"
+
+#: ../data/functions.xml.in.h:135
+msgid ""
+"Calculates the double factorial of an integer. Multiplies the argument with "
+"every second lesser positive integer (n(n-2)(n-4)...). Can also be entered "
+"as a number followed by two exclamation marks."
+msgstr ""
+"Вычисляет двойной факториал целого числа. Умножает аргумент на каждое "
+"уменьшенное на 2 положительное целое число (n (n-2) (n-3)...). Также можно "
```
Comment (i-ky) [inline po-defs/ru.po:3316]:
```suggestion
"уменьшенное на 2 положительное целое число (n (n-2) (n-4)...). Также можно "
```

File: po-defs/ru.po [hunk 9]
```diff
@@ -0,0 +1,11408 @@
...
+"argument with the factorial of every lesser positive integer (1! * 2! ... "
+"n!)."
+msgstr ""
+"Вычисляет суперфакториал целого числа. Умножает факториал аргумента на "
+"факториал каждого меньшего положительного целого числа (1! * 2! ... n!) "
+"(произведение первых n факториалов)."
+
+#: ../data/functions.xml.in.h:147
+msgid "Permutations (Variations)"
+msgstr "Размещение (варианты)"
```
Comment (i-ky) [inline po-defs/ru.po:3383]: I believe correct translation would be ["перестановки"](https://ru.wikipedia.org/wiki/Перестановка).

**Comment by idamir** [comment]
All items are translated.

**Pull request merged**